### DISCUSSION TITLE
v2.1.8

### DISCUSSION THREAD

**Discussion by edmondas** [poweradmin/poweradmin#405] [General]
## What's Changed
* dynamic_update: fix sql-query fetching records of zones by @florianschroen in https://github.com/poweradmin/poweradmin/pull/288
* add powerdns 4.0 record types and others by @bribroder in https://github.com/poweradmin/poweradmin/pull/291
* Add message to Japanese translation file in https://github.com/poweradmin/poweradmin/pull/309
* Maintenance update by @JeroenBo in https://github.com/poweradmin/poweradmin/pull/305
* mcrypt_ to openssl_ encrypt&decrypt  by @j4Hu in https://github.com/poweradmin/poweradmin/pull/337
* Check index and prevent notice by @SG5 in https://github.com/poweradmin/poweradmin/pull/330
* Add value for zone_templ_id by @LeePorte in https://github.com/poweradmin/poweradmin/pull/323
* PHP 7.3 + PCRE2 compatibility by @j4Hu in https://github.com/poweradmin/poweradmin/pull/360
* Spelling by @jsoref in https://github.com/poweradmin/poweradmin/pull/333
* update TLDs in toolkit by @frankhabicht in https://github.com/poweradmin/poweradmin/pull/357
* Support for new LUA resource records of PowerDNS Authoriative Server version 4.2 by @shinsterneck in https://github.com/poweradmin/poweradmin/pull/352
* Solve bug: "continue" targeting switch is equivalent to "break" by @BaseMax in https://github.com/poweradmin/poweradmin/pull/383
* Remove change_date as it's removed per pdns 4.2 by @KoffieNu in https://github.com/poweradmin/poweradmin/pull/395
* Update record.inc.php by @Niwreg in https://github.com/poweradmin/poweradmin/pull/398
* Incompatibility in PHP version 8.0 by @BaseMax in https://github.com/poweradmin/poweradmin/pull/391
* Add ldap authentication config option to add_user.php by @mark-wagner in https://github.com/poweradmin/poweradmin/pull/329
* In SQL, SUBSTR and SUBSTRING are used depending on the db_type by @nogisawa in https://github.com/poweradmin/poweradmin/pull/374
* DNSkey properly showing data with CSK type key by @zzplab in https://github.com/poweradmin/poweradmin/pull/292
* Add modern theme by @DecentM in https://github.com/poweradmin/poweradmin/pull/286
* Update SOA change_date in autoserial mode when deleting record by @jirka-grunt in https://github.com/poweradmin/poweradmin/pull/279
* Fix migration table field names in the field array by @ZeiP in https://github.com/poweradmin/poweradmin/pull/369

**Full Changelog**: https://github.com/poweradmin/poweradmin/compare/v2.1.7...v2.1.8

<hr /><em>This discussion was created from the release <a href='https://github.com/poweradmin/poweradmin/releases/tag/v2.1.8'>v2.1.8</a>.</em>

**Comment by edmondas** [comment]
Unfortunately I had no time to prepare the migration document, but you should generally follow the same pattern: create a backup of the files and database, remove old files (keep only the configuration file) then copy the new files.

**Comment by BaseMax** [comment]
Glad to have contributed to this growth. Although small

Happy coding **^_^**
